pandas/core/accessors.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+
+An example/recipe for creating a custom accessor.
```

Comment:
Just pushed commits that address some of these.  Took the example out of the module docstring, put a comment in PandasDelegate.__doc__ pointing readers towards the existing examples.